specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2023-08-02-preview/SpringEnvironments.json

```diff
@@ -0,0 +1,494 @@
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "SpringEnvironments_CheckNameAvailability": {
+            "$ref": "./examples/v2/SpringEnvironments/SpringEnvironments_CheckNameAvailability.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SpringEnvironmentsCollection": {
+      "description": "Collection of Spring Environments",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SpringEnvironment"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "SpringEnvironmentForPatchBody": {
+      "description": "An environment for hosting Azure Spring Apps only for patch body",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SpringEnvironment"
+        }
+      ],
+      "required": []
+    },
+    "SpringEnvironment": {
+      "description": "An environment for hosting Azure Spring Apps",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Spring environment resource specific properties",
+          "type": "object",
+          "properties": {
+            "provisioningState": {
+              "description": "Provisioning state of the Service",
+              "enum": [
+                "InProgress",
+                "Deleting",
+                "Deleted",
+                "Succeeded",
+                "Failed",
+                "Canceled"
+              ],
+              "type": "string",
+              "readOnly": true,
+              "x-ms-enum": {
+                "name": "ProvisioningState",
+                "modelAsString": true
+              }
+            },
+            "environmentId": {
+              "description": "The resource Id of the App Environment that the Spring Apps instance builds on",
+              "type": "string"
+            },
+            "infraResourceGroup": {
+              "description": "The name of the resource group that contains the infrastructure resources",
+              "type": "string"
+            },
+            "zoneRedundant": {
+              "description": "Whether or not this Spring Environment is zone-redundant.",
+              "enum": [
+                "true",
+                "false"
```

Comment:
Instead of values true and false, defnine these as something specifc e.g. "ZoneRedundant", "None" with definiton name being "zoneRedundancy". That will make it extensible in future for other redundancy options.